core/src/main/java/org/apache/accumulo/core/rpc/clients/TServerClient.java

```diff
@@ -54,6 +54,8 @@
 
 public interface TServerClient<C extends TServiceClient> {
 
+  static final String PREFERRED_HOST = "org.apache.accumulo.client.rpc.preferred.host";
```

Comment:
Not advocating for this, but was wondering if this could be an accumulo client property instead of a java system property.   The only advantage I can think of for this is that it makes documenting this easier in that we auto generate docs for the client props.

The advantage of the system prop is that it seems a bit easier to apply at runtime.

Reply:
I was just implementing the suggestion in the ticket. I'm good with it either way.
@ctubbsii  - this was your suggestion to make it a system property. Thoughts on client property?

Reply:
This kind of thing really seems like it's for troubleshooting, and not really something we'd want users to set in the normal case. It could be made a client property that can go in the user's client config file, but some of the reasons why I like it as a system property are that it's not treated like a "normal" configuration, and you have to know about it, and know what you're doing to use it. And for those cases, the fact that it is less well-documented, and treated like an internal debugging flag, seems to be in its favor. I really wouldn't want people setting this in their client config and then forgetting about it... like somebody who drives on the highway while forgetting to release the parking brake.